examples/wasm/iwasmt.c

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2019 Intel Corporation.  All rights reserved.
+ * Copyright (C) 2020 TU Bergakademie Freiberg Karl Fessel
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+//
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+
+#include <string.h>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstrict-prototypes"
+#include "wasm_export.h"
+#pragma GCC diagnostic pop
```

Comment:
This warning is generated if the forward declaration of a function looks like this `void foo()` instead of this `void foo(void)`.

Can't we just fix `wasm_export.h`, since this is a relatively trivial fix?

Reply:
that is an upstream discusion i allready had

Reply:
i did recheck this wasn't a issue that i didn't get upstream therefor i will upstream a fix 
https://github.com/bytecodealliance/wasm-micro-runtime/pull/331

Reply:
```suggestion
#include "wasm_export.h"
```

Github says the fix was merged upstream, so this should no longer be needed, right?